client/src/components/LogInForm/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { logIn } from '@/lib/api/user';
+import { Button } from '@/lib/design';
+import React, { useState } from 'react';
+import user from '@/recoil/user';
+import { useSetRecoilState } from 'recoil';
+import * as S from './style';
+import { taskSortByUpdate } from '@/lib/utils/sort';
+import { UserState } from '@/contexts/userContext';
+
+export const LogInForm = () => {
+  const setUserState = useSetRecoilState(user);
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const onChangeEmail = (e: React.ChangeEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    setEmail(e.target.value);
+  };
+  const onChangePassword = (e: React.ChangeEvent<HTMLInputElement>) => {
+    e.preventDefault();
+    setPassword(e.target.value);
+  };
```

Comment:
```suggestion
  const {value : email, onChange : onChangeEmail} = useInput('');
  const {value : password, onChange : onChangePassword} = useInput('');
```
useInput 커스텀 훅을 활용하면 이렇게 줄이실 수 있습니다!

Reply:
이 부분 오늘 해수님이 수정해주신건가요?